osquery/tables/system/posix/sudoers.cpp

```diff
@@ -87,6 +102,11 @@ void genSudoersFile(const std::string& filename,
       continue;
     }
 
+    // Check if a blackslash is the last character on this line.
+    if (!is_include && !is_includedir && line.at(line.size() - 1) == '\\') {
```

Comment:
Here and on line 69 above, `line.back()` would be a shorter more readable way to express `line.at(line.size() - 1)`

Reply:
Fixed